3.Tests/UI/Features/Item/DueDate.feature

```diff
@@ -41,16 +41,26 @@ Feature: Item Due Date
             | 1 month  | 20 Jul 12:00 PM |
             | 2 months | 20 Aug 12:00 PM |
 
+<<<<<<< HEAD
+    @Smoke @Regression @create.project.Errands @delete.projects @create.item.GoToGym @Due_Date_Item
+    Scenario Outline: Check if item appear on given section
+        When the user clicks on 'Project Button' <Errands> at 'Project Component'
+            And the user hovers on "Get Item" <GoToGym> at 'Items Component'
+            And the user clicks on 'Item DueDate' <GoToGym> at 'Items Component'
+            And inputs "<Date>" as due date
+        Then the "GoToGym" in "Errands" should be displayed in "<When>" section
+=======
     # @Smoke @Regression @delete.projects @create.project.Today @delete.projects @create.item.GoToGym @Due_Date_Item
     # Scenario Outline: Check if item appear on Today section
     #     When the user clicks on 'Project Button' <Today> at 'Project Component'
     #         And the user hovers on "Get Item" <GoToGym> at 'Items Component'
     #         And the user clicks on 'Item DueDate' <GoToGym> at 'Items Component'
     #         And inputs "<Date>" as due date
     #     Then the "GoToGym" should be displayed in "<When>" section
+>>>>>>> main
```

Comment:
Could you please fix the merging naming HEAD thingy

Reply:
Yeah, thanks for pointing this.